ddtrace/logger.py

```diff
@@ -0,0 +1,40 @@
+import logging
```

Comment:
This module should be internal `ddtrace/_logger.py`

Reply:
I'm going to mark this as resolved because it's `_logger.py` now.